project/urls.py

```diff
@@ -25,5 +25,5 @@
     path("calendar/", include("events.urls")),
     path("news/", include("news_app.urls")),
     path("chats/", include("chats.urls")),
-    path("admin/", admin.site.urls),
+    path('admin/', admin.site.urls),
```

Comment:
```suggestion
    path('admin/', admin.site.urls),
```
大家要不要考慮統一使用單引號或是雙引號？

Reply:
OKOk,我投雙引號一票

Reply:
雙引號可以。

Reply:
個人偏向雙引號 🐸

Reply:
雙引號++